codecov.yml

```diff
@@ -9,13 +9,19 @@ ignore:
   - "**/constants/**/*"
   - "constants/**/*"
 
-coverage:
-  status:
-    project:
-      default:
+flag_management:
+  default_rules:
+    statuses:
+      - type: project
         target: auto
         threshold: 1%
         if_ci_failed: error
-    patch:
-      default:
-        target: 80%
\ No newline at end of file
+      - type: patch
+        target: 80%
+  individual_flags:
+    - name: e2e-tests
+      statuses:
+        - type: project
+          target: auto
```

Comment:
do you need to repeat this from the default_rules?

Reply:
actually no